woocommerce/woocommerce-android
# Convert System Status Report Screen to Compose

### Description


This PR converts the System Status Report screen to use Jetpack Compose.

### Testing instructions


1. Open Settings > Help & Support > System Status Report.
2. Ensure that the contents and functionality there work as usual.
